logo_purple.svg

```diff
@@ -0,0 +1,1349 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   style="fill-rule:evenodd"
+   id="svg7046"
+   version="1.1"
+   viewBox="0 0 1350 1350"
+   height="1800"
+   width="1800">
```

Comment:
I have no idea. Inkscape did it probably. I don't know much about SVG.

The natural width is 1800 so I can change that. Wait for it...